src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2101,7 +2098,7 @@ namespace winrt::TerminalApp::implementation
 
         // Moving the pane from the current tab might close it, so get the next
         // tab before its index changes.
-        if (_tabs.Size() > tabIdx)
+        if (tabIdx < _tabs.Size())
```

Comment:
change in behavior?

Reply:
I don't _think_ so

Reply:
I don't know why I felt compelled to make it `small thing < big thing` in the true case. Just felt like a more sane bounds check here